src/compojure_throttle/core.clj

```diff
@@ -1,20 +1,29 @@
 (ns compojure-throttle.core
-  (:require [clojure.core.cache :as cache]
-            [environ.core :refer [env]]
+  (:require [clj.ip :refer [compile]]
```

Comment:
for readability, our guidelines suggest using `:as` and not `:refer` as `(ip/compile str)` gives us a little more context

Reply:
i was following the instructions on the lib page, but will change